_schemaV2.graphql

```diff
@@ -4108,6 +4108,11 @@ type CollectionsEdge {
   node: Collection
 }
 
+input CollectionsInput {
```

Comment:
Naming suggestion: The plural collection"s" here sounds a bit unintuitive with the singular data inside. Maybe we can call it `UpdateMeCollectionInput` so the mutation below reads a bit nicer:

```graphql
input updateMeCollectionsMutationInput {
  attributes: [UpdateMeCollectionInput!]!
  clientMutationId: String
}
```

(There is an existing [`updateCollectionInput`](https://github.com/artsy/metaphysics/blob/0f0cf799cd780744a9e2ce8ac87d7ca01c832c65/_schemaV2.graphql#L18868-L18874) that requires an id and a name. 🤷  The collection modeling for multiple purposes is kind of making naming tricky.)